static/css/zamboni/developers.css

```diff
@@ -904,16 +904,25 @@ pre.email_comment {
     background-repeat: no-repeat;
     padding: 2px 0 2px 24px;
 }
+.html-rtl #upload-status-results strong {
+    padding: 2px 24px 2px 0;
+}
 
 #upload-status-results.status-fail strong {
     color: #D81916;
     background-position: 0 -120px;
 }
+.html-rtl #upload-status-results.status-fail strong {
+    background-position: 100% -120px;
+}
 
 #upload-status-results.status-pass strong {
     color: #325487;
     background-position: 0 -80px;
 }
+.html-rtl #upload-status-results.status-pass strong {
```

Comment:
This could have been nested above and then you could have done:

```less
 #upload-status-results.status-pass strong {
     color: #325487;
     background-position: 0 -80px;

     .html-rtl & {
        background-position: 100% -80px;
     }
}
```